ccd-client/src/main/java/uk/gov/hmcts/cmc/ccd/client/StartCaseApi.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.cmc.ccd.client;
+
+import org.springframework.cloud.netflix.feign.FeignClient;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import uk.gov.hmcts.cmc.ccd.client.model.StartEventResponse;
+
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+
+@FeignClient(name = "core-case-data-api", url = "${core_case_data.api.url}")
+public interface StartCaseApi {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Move constants to a class or enum. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS1214)